backend/stream/utils/db_util.py

```diff
@@ -0,0 +1,72 @@
+from sqlalchemy import create_engine, Column, Float, JSON, String
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from backend.stream.utils import file_util
+
+
+def update_article_url_db(article_dicts, publisher, db_file_path):
+    file_util.make_parent_dirs(db_file_path)
+    base_cls = declarative_base()
+
+    class Article(base_cls):
+        __tablename__ = publisher
+
+        url = Column(String, primary_key=True)
+        title = Column(String)
+        publishedAt = Column(String)
+        addedAt = Column(String)
+
+    engine = create_engine('sqlite:///{}'.format(db_file_path), echo=True)
```

Comment:
Decouple the engine. This way you can use a test db (instead of the prod db) during testing. Also, I'd consider allowing the user to set  `echo=false` since these messages can get kind of annoying when you aren't debugging.

Reply:
Done